src/domains/recipes/recipe.service.ts

```diff
@@ -91,4 +93,15 @@ export class RecipeService implements IRecipeService {
 
     return newRecipe;
   }
+
+  async deleteRecipe(recipeId: number, userId: number) {
+    const recipe = await this.recipeRepository.getRecipeById(recipeId);
+    await this.recipeToIngredientRepository.deleteFromDatabase(recipe.id);
```

Comment:
```suggestion
    await this.recipeToIngredientRepository.delete(recipe.id);
```
Du behöver inte ha med `...FromDatabase`.